### ISSUE TITLE
Restrict swipe gesture to a certain area

### ISSUE THREAD

**Issue by suprb** [fortmarek/SwipeViewController#16]
Is there any way to restrict the  pan/swipe to a certain area? I'd like to leave the bottom part of my view as an area where it ignores the gesture. Is it doable?

**Comment by fortmarek** [comment]
Since SwipeViewController subclasses UIPageViewController it should be possible. Try looking at this code: https://gist.github.com/TosinAF/805a22ab5a3e4ffe17a8/

**Comment by relaxsus** [comment]
Okay, but if I need tableView gestures to work? How realize it?

**Comment by fortmarek** [comment]
I have not tried this myself, but please do try to use the gist I shared above. You can implement part of it in `SwipeViewController.swift` `syncScrollView()` function. Any PRs are welcome 🤓

**Comment by relaxsus** [comment]
In SwipeViewController.swift
```swift
    open var isScrolling = true
```
inside func setPageController()
```swift
    pageController.delegate = self
    if isScrolling {
            pageController.dataSource = self
    }
```

In YourViewController: SwipeViewController where you want to disable swipe
```swift
    isScrolling = false
```

**Comment by fortmarek** [comment]
Hmm, interesting solution, I will take a look at it tomorrow. This is for disabling swiping for one `ViewController` as a whole, and not a part of it, but still could be useful for some people out there! 🎊

**Comment by relaxsus** [comment]
Yes, it is for disabling the whole ViewController. I needed just it:)

**Comment by SeasiaInhouse** [comment]
What if I want to disable swiping in ViewController: UIViewController not ViewController: SwipeViewController

**Comment by fortmarek** [comment]
@SeasiaInhouse, could you please describe your problem more thoroughly? Thanks.
